test/vsphere/vsphere-single-master.yaml

```diff
@@ -0,0 +1,52 @@
+##
```

Comment:
Not necessarily for this PR, but most of these comments should probably find their way into the godoc comments in types.go so that they pull through to https://karina.docs.flanksource.com/reference/config/

Likewise with the validations

Reply:
I've added with latest commit.

( :smile: needed an update to retrigger *test-next*)